fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -177,16 +177,23 @@ class ThresholdOptimizer(BaseEstimator, MetaEstimatorMixin):
         Defines which method of the ``estimator`` is used to get the output
         values.
 
-        - 'auto': use one of ``predict_proba``, ``decision_function``, or
-          ``predict``, in that order.
-        - 'predict_proba': use the second column from the output of
-          `predict_proba`. It is assumed that the second column represents the
-          positive outcome.
-        - 'decision_function': use the raw values given by the
-          `decision_function`.
-        - 'predict': use the hard values reported by the `predict` method if
-          estimator is a classifier, and the regression values if estimator is
-          a regressor. This is equivalent to what is done in [1]_.
+            'auto'
+                use one of ``predict_proba``, ``decision_function``, or 
+                ``predict``, in that order.
+            
+            'predict_proba'
+                use the second column from the output of `predict_proba`. 
+                It is assumed that the second column represents the positive 
+                outcome.
+            
+            'decision_function'
+                use the raw values given by the `decision_function`.
```

Comment:
sure, we'll

Reply:
Actually, I'm not really sure what you meant by :code: before decision or predict because when i add they are not making any sense to me. let me show you 
1) 
![image](https://user-images.githubusercontent.com/69182153/212239393-5a4c930b-c2a5-4f73-b6de-a11486217f54.png)

results:

![image](https://user-images.githubusercontent.com/69182153/212238979-85daf4ce-1835-4446-9271-c8b93689e3b9.png)

2) 
![image](https://user-images.githubusercontent.com/69182153/212239524-c179ab33-9f96-48c6-ba15-2ead6b904fb9.png)


results:

![image](https://user-images.githubusercontent.com/69182153/212239116-2f6ba55c-45b2-4c66-a085-2e9cba28d398.png)

let me know you if there's something you might be interested or what else you intended?

Reply:
Try removing the space in between :code: and the backquote. It's just a way of telling sphinx how to render the content of the back quotes.

Reply:
you mean this?
![image](https://user-images.githubusercontent.com/69182153/212285070-9a805e11-5fc0-445f-912f-6b0ed42e7857.png)

Reply:
if so it doesn't work either.
![image](https://user-images.githubusercontent.com/69182153/212288838-958de286-a9a6-4ad5-b365-bb649126ea93.png)

I have searched the sphinx documentation and stackoverflow, didn't find anything helpful.

Reply:
can you please let me know the exact behavior you want, so i search for that?

Reply:
Backquote is the backtick character **"`"**

So you will have to do something like this - 
```
:code:`decision_function`
```

Reply:
thanks.